packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -76,15 +85,47 @@ export interface EditableTree {
     readonly [proxyTargetSymbol]: object;
 
     /**
-     * Anchor to this node.
-     * Valid as long as this EditableTree's context is not freed.
-     * Might not point to any node if this node is deleted from the document.
-     *
-     * TODO: When a proper editing API is exposed on EditableTree directly,
-     * this should become an implementation detail and rbe removed from this API surface.
+     * Creates a child of this node.
+     */
+    readonly [insertNodeSymbol]: (key: string, value: ITreeCursor) => boolean;
```

Comment:
Should document return value. Don't use a plain string for field keys (see set value comment for details), unclear what this does if its a value or optional field.